genesyscloud/architect_flow/data_source_genesyscloud_flow.go

```diff
@@ -22,9 +22,9 @@ func dataSourceFlowRead(ctx context.Context, d *schema.ResourceData, m interface
 	// Query flow by name. Retry in case search has not yet indexed the flow.
 	return util.WithRetries(ctx, 5*time.Second, func() *retry.RetryError {
 		for pageNum := 1; ; pageNum++ {
-			flows, _, getErr := p.GetAllFlows(ctx)
+			flows, resp, getErr := p.GetAllFlows(ctx)
 			if getErr != nil {
-				return retry.NonRetryableError(fmt.Errorf("error requesting flow %s: %s", name, getErr))
+				return retry.NonRetryableError(fmt.Errorf("Error requesting flow %s: %s %s", name, getErr, resp))
```

Comment:
For this line, I see the warning "Error string should not be capitalized or end with punctuation mark" on GoLand. It's fairly non-consequential, but better to keep warnings to a minimum at the same time

Reply:
There's no need to change all the error messages in this PR to lower case, just saying this as a heads up that changing from lower to upper is the wrong way to go